src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -261,6 +242,11 @@ public void storeContext(Context context, HttpServletRequest request) {
 		HttpServletResponse response = vtc.getResponse();
 		HttpSession session= request.getSession(true);
 		M_log.debug("session id: "+session.getId());
+		boolean admin = isAccountAdmin(request);
+		if (admin) {
+			M_log.info(String.format("The user \"%s\" is account admin in the course %s",
+					request.getParameter(CUSTOM_CANVAS_USER_LOGIN_ID), request.getParameter(CUSTOM_CANVAS_COURSE_ID)));
```

Comment:
Is there a better name than "CUSTOM_CANVAS_".... I can't tell the purpose of the parameter just by looking at the name.